script/Ciel/Day 8/08_01_CIEL08_4.txt

```diff
@@ -298,27 +325,27 @@ Once, it resulted in an act, driven by sexual desire, cruel and all-too gruesome
 [sha:ad4b2007edef7aeec2787a777396192a32b97e60]{
 -- Page 357, Offset 19149.
 -- 頭を振って残像を振り払う。
-Shaking my head, I brush off those afterimages.
+I shake my head and chase away those ghostly images.
```

Comment:
mm, maybe this is just the limits of my japanese, but I think "ghostly images" is a little too editorialized, no? Instead of it being like "impression of the past", it might be interpreted as "horrific" or "terrifying" scene